galaxyproject/galaxy
# [23.1] fix font-weight disparity between browsers

- Confirming that this issue, #16325 , was happening on Chrome browsers. 
- For Chrome browsers, specify font-smoothing to ensure font-weight parameter is distinct from normal text. Especially visible in the Tool Form pages (Tools) where `<P>aragraph` and `<Strong>` tags are widely used.
- Below screenshot example is from the "Filter" tool Tool Form page. 

![screeenshot_github_issue_16325_Antialiased](https://github.com/galaxyproject/galaxy/assets/3672779/e70904ef-794a-44eb-8561-40e5b33df08b)

![screenshot_AFTER_v2](https://github.com/galaxyproject/galaxy/assets/3672779/a57a374a-922e-41b3-9819-fcac381d2880)

![screenshot_Tools_Menu_before_after](https://github.com/galaxyproject/galaxy/assets/3672779/8157d22c-cc5f-4753-adc6-dc4ec100cda7)

## How to test the changes?
(Select all options that apply)
  1. In the "Tools" menu, select the "Filter" tool
  2. The font with "font-weight" parameter with `bolder` value should clearly appear distinct from `normal` value text

## License
